src/actions/index.js

```diff
@@ -45,10 +47,8 @@ export function getUserInfo() {
     getUserDetails().then((res) => {
       // console.log(res.data);
       dispatch(success(res.data));
-    }, (err) => {
-      console.error(err);
     }).catch((err) => {
-      console.error(err);
+      Raven.captureException(err, sentryExtra('Error while fetching user Info'));
```

Comment:
How are you communicating this error to the user?
When the getUserInfo API call fails, how are you intimating the user that 'data retrieval' from the server is failed? 

You are registering the exceptions with Raven which is just going to track the exception on Sentry Dashboard. But, it seems nowhere we are showing the error message to the user